pkg/engine/planner/physical/optimizer.go

```diff
@@ -198,26 +199,54 @@ func (r *projectionPushdown) apply(node Node) bool {
 		// Always project timestamp column
 		projections[len(node.PartitionBy)] = &ColumnExpr{Ref: types.ColumnRef{Column: types.ColumnNameBuiltinTimestamp, Type: types.ColumnTypeBuiltin}}
 
-		return r.applyProjectionPushdown(node, projections, false)
+		ambiguousProjections := collectAmbiguousColumns(node.PartitionBy)
+		return r.applyProjectionPushdown(node, projections, ambiguousProjections, false)
+	case *VectorAggregation:
```

Comment:
cc @ashwanthgoli Is there a reason we didn't have pushdown for VectorAggregation before?

Reply:
Ah, okay. Do you have a suggestion for what we should do in the projection pushdown for the parse?